service/controller.go

```diff
@@ -2091,6 +2093,10 @@ func (s *service) getCapacityForAllSystems(ctx context.Context, protectionDomain
 	return capacity, nil
 }
 
+var maxVolumesSizeForArray = make(map[string]int64)
```

Comment:
Will it be okay to add one line comment about this var?
Something like: // maxVolumesSizeForArray -  store the maxVolumesSizeForArray

Reply:
@adarsh-dell review comments addressed